objects/bee103/script.lua

```diff
@@ -381,6 +381,7 @@ end
 function addFear()
     local fearPool = Global.getVar("fearPool")
     local generatedFear = Global.getVar("generatedFear")
+    if generatedFear == 0 and fearPool == 0 then return end
```

Comment:
with my latest changes checking to see if the game has started, this shouldn't be needed anymore

Reply:
Which changes are these?

Reply:
https://github.com/spirit-island/spirit-island-mod/commit/06f1927f110a57e57ab19621fc52d8468c750f52#diff-5ab2a5ad8146be10d245f73262559ff9340e25d4d8a4d3426b4e1e17a914de72

Reply:
I'll remove these when I pull latest beta back in.  It'd be part of the conflict resolution anyways.